code/src/index.css

```diff
@@ -1,13 +1,164 @@
-body {
+* {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-image: url(img/holiday.jpg);
+  background-size: 100% 100%;
```

Comment:
Small detail, but maybe try background-size:cover; to prevent background image from "merging" when viewing on smaller devices?

Reply:
great tips!